components/brave_account/mojom/brave_account_settings_handler.mojom

```diff
@@ -5,6 +5,22 @@
 
 module brave_account.mojom;
 
-interface BraveAccountSettingsHandler {
+enum AccountState {
+  kLoggedOut,
+  kVerification,
+  kLoggedIn,
+};
+
+interface BraveAccountRow {
+  UpdateState(AccountState state);
+};
+
+interface BraveAccountRowHandler {
+  GetAccountState() => (AccountState state);
   OpenDialog();
 };
+
+interface BraveAccountSettingsHandler {
+  CreateRowHandler(pending_remote<BraveAccountRow> row,
+                   pending_receiver<BraveAccountRowHandler> row_handler);
```

Comment:
I don't understand why both these methods couldn't live here? Now this interface only exists to bind a row handler (and both methods on that seem like they belong here)